grammar.js

```diff
@@ -30,7 +30,7 @@ module.exports = grammar({
     extras: $ => [
         $.BLOCK_COMMENT,
         $.LINE_COMMENT,
-        $.SPACE
+        $._SPACE
```

Comment:
This is the only change to the grammar itself.
@adrpo Will this break tools downstream?
Otherwise I don't see a point in showing whitespace in the tree.